performance_test/src/experiment_configuration/experiment_configuration.cpp

```diff
@@ -202,7 +201,8 @@ void ExperimentConfiguration::setup(int argc, char ** argv)
     }
     if (vm.count("use_drive_px_rt")) {
       // Magic numbers for the drive px.
-      proc_rt_init(62, 5);
+      pre_proc_rt_init(62, 5);
```

Comment:
If you are on it can you better describe what those numbers mean?

Reply:
Put a better description